Content.Shared/_ES/Masks/Traitor/Events/ESAddMaskOnUseEvents.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Content.Shared.DoAfter;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared._ES.Masks.Traitor.Events;
+
+[Serializable, NetSerializable]
+public sealed partial class ESAddMaskOnUseDoAfterEvent : DoAfterEvent
+{
+    public override DoAfterEvent Clone() => this;
```

Comment:
This does not clone the event when it should.

Reply:
I genuinely don't know why but this is basically what is done for every single `DoAfterEvent` (which doesn't really make sense) but is otherwise fine? I guess?